doc/introduction/configuration.rst

```diff
@@ -52,4 +52,35 @@ Configuration options
 
 **port (int)** (*optional*)
     The port to be used when connecting to the remote service.
-    Corresponding environment variable: ``SF_API_PORT``
\ No newline at end of file
+    Corresponding environment variable: ``SF_API_PORT``
+
+Store your account
+------------------
+
+Using the :func:`~.store_account` function, a configuration file can be created easily. It only requires specifying the authentication token. Apart from that, further configuration options can be passed as keyword arguments.
+
+Configure for the current SF project
+************************************
+
+The following is an example for using ``store_account`` with defaults:
+
+.. code-block:: python
+
+    import strawberryfields as sf
+    sf.store_account("MyToken")
+
+where ``"MyToken"`` contains the user specific authentication token.
+
+It is advised to execute this code snippet **only once** per configuration. This should be done in the same directory where the SF project can be found, separately from executing any other Strawberry Fields scripts. Using the default options it will store the account in the *current working directory* by creating a ``config.toml`` file.
```

Comment:
Will a user know what "per configuration" means? Suggest making a bit more clear

Reply:
What happens if the user runs this code snippet twice? Does anything break?

Reply:
Sure, I'll reword it :slightly_smiling_face: Great question, it simply creates the same file each time (so upon the second call, the first file will be overwritten)

The reason why I've included it is not to encourage users to have them in their scripts and also mainly to use this once and then in most cases not really bother about configuration at all. For other cases they could delve deeper into understanding the different options.

Reply:
It is worth noting that after a discussion with @josh146, I've now updated this such that the "user_config" option (that is *global*) is the default. The behaviour would be the same nonetheless, the file would be overwritten.